From: David Herrmann <dh.herrmann@gmail.com>
Date: Mon, 16 Mar 2015 10:17:13 +0100
Subject: [PATCH] selftests/kdbus: fix gitignore

Drop unused elements from .gitignore (which are leftovers when
documentation was placed in the same directory).
Add "kdbus-test" to .gitignore, which is the test binary of all kdbus
selftests.

Signed-off-by: David Herrmann <dh.herrmann@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 tools/testing/selftests/kdbus/.gitignore | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/tools/testing/selftests/kdbus/.gitignore b/tools/testing/selftests/kdbus/.gitignore
index 7b421f76c888..d3ef42f6ada6 100644
--- a/tools/testing/selftests/kdbus/.gitignore
+++ b/tools/testing/selftests/kdbus/.gitignore
@@ -1,3 +1 @@
-*.7
-manpage.*
-*.proc
+kdbus-test
